fixtures/cameo/nanospot-120.json

```diff
@@ -139,53 +135,90 @@
     },
     "Program": {
       "type": "Effect",
+      "defaultValue": 0,
```

Comment:
That's true and the capability should keep its name. But most plugins will implement the last channel in `11ch` as a `No function` channel, 'cause it's the default. So the `No function` capability must switch to `Program Speed` (which has no effect, sure) but the plugins will implement the last channel as a `Speed` channel.